src/ssvc/dp_groups/coordinator_publication.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+"""
+file: coordinator_publication
+author: adh
+created_at: 9/21/23 11:40 AM
+"""
+from ssvc.decision_points.exploitation import EXPLOITATION_1
+from ssvc.decision_points.public_value_added import PUBLIC_VALUE_ADDED_1
+from ssvc.decision_points.supplier_involvement import SUPPLIER_INVOLVEMENT_1
+from ssvc.dp_groups.base import SsvcDecisionPointGroup
+
+
+COORDINATOR_PUBLICATION_1 = SsvcDecisionPointGroup(
+    name="Coordinator Publication",
```

Comment:
Comment from @sei-vsarvepalli:

>The Decision Point Group should not have any of the following fields - it is only a collection of decision points. These seem like they belong in the Decision Tree and not in a Decision Point Group as I understand from our discussions.

Reply:
Per conversation yesterday, I refactored another mixin class `_Keyed` that is used by Decision Points and Decision Point Values but _not_ Decision Point Groups in https://github.com/CERTCC/SSVC/pull/328/commits/57609f1a8b7f7c597576b2fb9cd48685b22fa6a9
then removed the "key" assignments from the DP group objects in
https://github.com/CERTCC/SSVC/pull/328/commits/63a3ca11ccf4e71cc10566d238620eaf89d64793

`DecisionPointGroup` objects only have `name`, `description`, and `version` now. I think that tracks what we discussed, but let me know if I've missed something.

Reply:
See comments beginning with https://github.com/CERTCC/SSVC/pull/328#discussion_r1352876222 below for specific changes to which classes have keys.